src/components/PoolingWidget/DefineSpread.tsx

```diff
@@ -1,52 +1,48 @@
-import React from 'react'
-import { TokenDetails } from '@gnosis.pm/dex-js'
+import React, { useCallback } from 'react'
 
-import { SpreadInformationWrapper, DefineSpreadWrapper } from './DefineSpread.styled'
+import { DefineSpreadWrapper } from './DefineSpread.styled'
 
-import { DEFAULT_DECIMALS } from 'const'
+import InputWithTooltip from '../InputWithTooltip'
 
-interface DefineSpreadProps {
-  selectedTokensMap: Map<number, TokenDetails>
+import { useFormContext, FieldError } from 'react-hook-form'
 
-  spread: number
-  setSpread: React.Dispatch<React.SetStateAction<number>>
+function formatErrorMessage(errorString: string): string {
+  const cleanedString = errorString.replace(/"/g, '')
+  return cleanedString[0].toUpperCase() + cleanedString.slice(1)
 }
 
-type SpreadInformationProps = Omit<DefineSpreadProps, 'setSpread'>
-
-export const SpreadInformation: React.FC<SpreadInformationProps> = ({ selectedTokensMap, spread }) => {
-  const tokenSymbolsString = React.useMemo(() => Array.from(selectedTokensMap.values()).map(token => token.symbol), [
-    selectedTokensMap,
-  ])
-
-  return (
-    <SpreadInformationWrapper>
-      <strong>Sell Spread</strong>
-      <p>
-        {tokenSymbolsString.join(', ')} for <b>at least</b> <br />
-        <i>${(1 + spread / 100).toFixed(DEFAULT_DECIMALS)}</i>
-      </p>
-      <strong>Buy Spread</strong>
-      <p>
-        {tokenSymbolsString.join(', ')} for <b>at most</b> <br />
-        <i>${(1 - spread / 100).toFixed(DEFAULT_DECIMALS)}</i>
-      </p>
-    </SpreadInformationWrapper>
-  )
+interface DefineSpreadProps {
+  isSubmitting: boolean
+  spread: number
 }
 
-const DefineSpread: React.FC<DefineSpreadProps> = ({ selectedTokensMap, spread, setSpread }) => {
-  const handleSpreadChange = ({ target: { value } }: React.ChangeEvent<HTMLInputElement>): void => {
-    if (+value < 0 || +value >= 100) return
+const DefineSpread: React.FC<DefineSpreadProps> = ({ isSubmitting }) => {
+  const { errors, register } = useFormContext()
 
-    setSpread(+value)
-  }
+  const onKeyPress = useCallback((event: React.KeyboardEvent<HTMLInputElement>): void => {
+    if (event.key === 'Enter') {
```

Comment:
Can I haz enter to submit form?

Reply:
I still can't `enter` on spread :(

Reply:
works for me on FF, whatre u using

Reply:
the PR URL hasnt updated due to the develop mismatch, ill merge and then it should work

Reply:
Working now \o/ yay!